pallets/thea-message-handler/src/lib.rs

```diff
@@ -27,18 +27,27 @@
 
 use frame_support::{pallet_prelude::*, traits::Get, BoundedVec, Parameter};
 use frame_system::pallet_prelude::*;
+pub use pallet::*;
 use parity_scale_codec::{Encode, MaxEncodedLen};
+use polkadex_primitives::utils::return_set_bits;
 use sp_runtime::{
 	traits::{BlockNumberProvider, Member},
 	transaction_validity::{InvalidTransaction, TransactionValidity, ValidTransaction},
 	RuntimeAppPublic, SaturatedConversion,
 };
 use sp_std::prelude::*;
-
-pub use pallet::*;
-use polkadex_primitives::utils::return_set_bits;
 use thea_primitives::{types::Message, Network, ValidatorSet};
 
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+
+pub trait WeightInfo {
+	fn insert_authorities(_b: u32) -> Weight;
+	fn incoming_message() -> Weight;
+	fn update_incoming_nonce(_b: u32) -> Weight;
+}
```

Comment:
Adding required empty line.
```suggestion
}

```